spharpy/spherical.py

```diff
@@ -88,6 +88,40 @@ def nm2acn(n, m):
     return n**2 + n + m
 
 
+def nm2fuma(n, m):
```

Comment:
Didn't we decide to change the naming conventions for this kind of function to `nm_to_fuma` to make conversion functions involving the normalizations more readable?